src/background/background.js

```
@@ -99,8 +99,41 @@ function handleBML(action) {
+                         chrome.tabs.sendMessage(tabId, {
+                             greeting: "load",
+                             code: result.currentBML
+                         });
+                     } else {
+                         console.warn("No BML code found in storage to load");
+                     }
+                 });
+             } else {
+                 // For unloading BML, we need to get the code from the editor and save it
+                 chrome.tabs.sendMessage(tabId, { greeting: "unload" }, function(response) {
+                     if (response && response.code) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)